app/code/Magento/CatalogGraphQl/Model/Resolver/Product/MediaGallery/Url.php

```diff
@@ -55,7 +55,7 @@ public function resolve(
         $context,
         ResolveInfo $info,
         array $value = null,
-        array $args = null
+        array $args = []
```

Comment:
Hi @tdgroot. We are introducing a major issue by this change since we are making the resolver implementation incompatible with the resolver interface. We need to find another way to handle it.